core/src/main/scala/org/ensime/indexer/ClassfileIndexer.scala

```diff
@@ -12,11 +13,8 @@ import org.objectweb.asm._
 import org.objectweb.asm.Opcodes._
 
 final class ClassfileIndexer(file: FileObject) extends SLF4JLogging {
```

Comment:
a lot of these changes could be raised against 2.0 without the graph. Might be an option if the performance issue drags on for much longer...